image/templates/helm/stackrox-secured-cluster/README.md.htpl

```diff
@@ -287,14 +287,9 @@ chart and their default values:
 |`collector.forceCollectionMethod`|Deprecated. This value has no effect. |`false` |
 |`collector.collectionMethod`|Either `CORE_BPF`, or `NO_COLLECTION`. |`CORE_BPF` |
 |`collector.disableTaintTolerations`|If you specify `false`, tolerations are applied to collector, and the collector pods can schedule onto all nodes with taints. If you specify it as `true`, no tolerations are applied, and the collector pods won't scheduled onto nodes with taints. |`false` |
-|`admissionControl.listenOnCreates`| This setting controls whether the cluster is configured to contact the StackRox Kubernetes Security Platform with `AdmissionReview` requests for `create` events on Kubernetes objects. |`false` |
-|`admissionControl.listenOnUpdates`|This setting controls whether the cluster is configured to contact the StackRox Kubernetes Security Platform with `AdmissionReview` requests for `update` events on Kubernetes objects.|`false` |
-|`admissionControl.listenOnEvents`|This setting controls whether the cluster is configured to contact the StackRox Kubernetes Security Platform with `AdmissionReview` requests for `update` Kubernetes events like `exec` and `portforward`.|`false` on OpenShift, `true` otherwise.|
-|`admissionControl.dynamic.enforceOnCreates`| It controls whether the StackRox Kubernetes Security Platform evaluates policies; if it’s disabled, all `AdmissionReview` requests are automatically accepted. You must specify `listenOnCreates` as `true` for this to work. |`false` |
-|`admissionControl.dynamic.enforceOnUpdates`| It controls whether the StackRox Kubernetes Security Platform evaluates policies for object updates; if it’s disabled, all `AdmissionReview` requests are automatically accepted. You must specify `listenOnUpdates` as `true` for this to work. |`false`|
-|`admissionControl.dynamic.scanInline`| |`false` |
+|`admissionControl.enforce`| This setting controls whether the StackRox Kubernetes Security Platform enforces policies. |`true` |
+|`admissionControl.failurePolicy`| This setting controls the failure policy for the ValidatingWebhookConfiguration. Can be "Ignore" or "Fail". |"Ignore" |
```

Comment:
Based on the other fields and their docs, I think we might need to explain what happens if the user chooses Fail or Ignore? Or link to the docs like the disable bypass annotations one.